dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -17,6 +17,9 @@ export const enhanceCollections = (
 			curated: enhanceCards(collection.curated, containerPalette),
 			backfill: enhanceCards(collection.backfill, containerPalette),
 			treats: enhanceCards(collection.treats, containerPalette),
+			config: {
```

Comment:
Is it worth expanding this with a note on other configuration options we might anticipate inside this object? My guess is that you do have some in mind, as you didn't surface this on a higher level